cookbook/session/locale_sticky_session.rst

```diff
@@ -116,10 +116,10 @@ before the ``FirewallListener``, which is responsible for handling authenticatio
 setting the user token on the ``TokenStorage``, you have no access to the user
 which is logged in.
 
-Let's pretend you have defined a ``locale"`` property on your ``User`` entity
-and you want to use this as the locale for the given user. To accomplish this,
-you can hook into the login process and update the user's session with the
-this locale value before they are redirected to their first page.
+Assumed you have defined a ``locale`` property on your ``User`` entity and
```

Comment:
Everything else looks perfect - so I made this last change at sha: 45aff1dbb054187c34aec7d5a17f20d5b8ee40a5

Thanks Christian!